README.md

```diff
@@ -41,13 +41,32 @@ Formats the token amount.
 #### Parameters
 
 - `options.symbol`: the token symbol, as a `String`. Overrides the value set in the constructor.
+- `options.commify`: whether the token amount should have comma separators
 - `options.digits`: the number of digits to display. Defaults to `2`.
 - `options.displaySign`: whether the sign (`-` or `+`) should be displayed for the amount.
 
 ### TokenAmount#format(options)
 
 Alias to TokenAmount#format().
 
+### TokenAmount#convert(rate, decimals, options)
+
+Converts from a rate, returning a new `TokenAmount` instance with the desired decimals and set
+options.
+
+#### Parameters
+
+- `rate`: the rate to convert from, as a `BigInt`, `String`, `Number` or `TokenAmount`.
```

Comment:
Oh interesting, I wouldn't have expected a `TokenAmount`!

Thinking about this, "convert 30 ANT with a rate of 30 ETH" doesn't make much sense 😄

Reply:
Haha that came from an offline conversation with @bpierre; we thought it might be useful.